spec/src/main/asciidoc/messaging-spec.adoc

```diff
@@ -478,7 +478,7 @@ messaging |Domain-specific API for pub/sub messaging
 
 |Session | |QueueSession |TopicSession
 
-|MessageProducer |JMSProducer |QueueSender |QueueReceiver
+|MessageProducer |JMSProducer |QueueSender |TopicPublisher
```

Comment:
I think it still misses separate line for consumers. What about this:

```suggestion
|MessageProducer |JMSProducer |QueueSender |TopicPublisher
|MessageConsumer |JMSConsumer |QueueReceiver |TopicSubscriber
```

Reply:
With added consumers:

![image](https://github.com/jakartaee/messaging/assets/11896137/51d6cd43-a050-4bc0-979f-faf47492f896)

Reply:
Perfect, thanks!